src/components/Header.tsx

```diff
@@ -0,0 +1,90 @@
+import logo from "@/assets/logo.png";
+import { BsPerson } from "react-icons/bs";
+import { GoSearch } from "react-icons/go";
+import { IoCartOutline } from "react-icons/io5";
+import { atom, useAtom } from "jotai";
+import searchResult from "@/components/SearchResult";
+import { showHide } from "@/components/AtomsConfig";
+
+export const searchValue = atom("");
+
+const Header = ({className}) => {
+  const [search, setSearch] = useAtom(searchValue);
+  const [show, setShow] = useAtom(showHide);
+
+  const headerArr = [
+    "Features",
+    "Shop",
+    "Doorbells",
+    "Security",
+    "Accessories",
+    "Journal",
+    "Our Story",
+    "Vendors",
+    "Contact",
+  ];
+
+  const firstLinks = headerArr.slice(0, 6);
+  const secondLinks = headerArr.slice(6);
+
+  const viewCard = () => {
+    setShow(!show);
+  };
+
+
+  return (
+    <div className={`w-full bg-[#f5f5f5] sticky top-0 z-10 ${className}`}>
+      <div className="flex flex-col gap-3 w-[1280px] mx-auto pt-[10px] ">
+        <div className=" flex justify-between items-center ">
+          <img src={logo} alt="" className="h-6" />
+          <div className="flex items-center gap-3">
+            <div className="relative">
+              <input
+                type="text"
+                placeholder="Search for anything"
+                className="border-2 text-sm text-left pr-[140px] py-[11px] rounded-md placeholder-gray-600 px-5 bg-[#e7e8ea]
+              focus:bg-white"
+                onChange={(e) => setSearch(e.target.value)}
+              />
+              <GoSearch className="absolute top-3 right-3 text-xl cursor-pointer" />
+              {Boolean(search.length)&& 
+                <div className="absolute w-full h-[240px] z-10 ">
+                  <searchResult search={search}/>
+                </div>
+              }
+            </div>
+            <BsPerson className="text-2xl cursor-pointer" />
+
+            <IoCartOutline
+              className="text-2xl cursor-pointer"
+              onClick={() => viewCard()}/>
+          </div>
+        </div>
+
+        <nav className="pb-5 flex justify-between">
+          <div className="inline-flex gap-10">
+            {firstLinks.map((data, index) => (
+              <p
+                key={index}
```

Comment:
do not use array index as key, learn why it is dangerous https://www.youtube.com/watch?v=xlPxnc5uUPQ